airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/bigquery/test_primary_key_streams/models/generated/airbyte_ctes/test_normalization/conflict_stream_name_conflict_stream_name_ab1.sql

```diff
@@ -0,0 +1,10 @@
+{{ config(schema="_airbyte_test_normalization", tags=["nested-intermediate"]) }}
+-- SQL model to parse JSON blob stored in a single column and extract into separated field columns as described by the JSON Schema
+select
+    _airbyte_conflict_stream_name_hashid,
+    {{ json_extract('conflict_stream_name', ['conflict_stream_name']) }} as conflict_stream_name,
```

Comment:
@ChristopheDuong here

Reply:
{"conflict_name": {"conflict_name": "hi"}}

Reply:
if somehow the macro would translate into something like `json_extract('conflict_stream_name', ['conflict_stream_name', 'conflict_stream_name'])`

maybe that would make it less ambiguous for the SQL engine (Destination)?